samples/subsys/usb/uvc/README.rst

```diff
@@ -0,0 +1,186 @@
+.. zephyr:code-sample:: uvc
+   :name: USB Video sample
+   :relevant-api: usbd_api video_interface
+
+   Send video frames over USB.
+
+Overview
+********
+
+This sample demonstrates how to use a USB Video Class instance to
+send video data over USB.
+
+Upon connection, a video device will show-up on the host,
+usable like a regular webcam device.
+
+Any software on the host can then access the video stream as a local video source.
+
+Requirements
+************
+
+This sample uses the new USB device stack and requires the USB device
+controller ported to the :ref:`udc_api`.
+
+Building and Running
+********************
+
+If a board is equipped with a supported image sensor, and ``zephyr,camera``
+node is chosen for the board, it will be used as the video source.
+The sample can be built as follows:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/subsys/usb/uvc
+   :board: arduino_nicla_vision/stm32h747xx/m7
+   :goals: build flash
+   :compact:
+
+If not, it is possible to test with a test pattern generator by using the
+``video-sw-generator`` snippet with any board:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/subsys/usb/uvc
+   :board: frdm_mcxn947/mcxn947/cpu0
+   :snippets: video-sw-generator
+   :goals: build flash
+   :compact:
```

Comment:
Please place this information at the beginning of the `Building and Running` chapter to make it clear that no special hardware is required to try it out.